src/Sylius/Bundle/TaxonomyBundle/Tests/Functional/app/AppKernel.php

```diff
@@ -42,18 +41,22 @@ public function registerBundles()
      */
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
-        $loader->load(__DIR__.'/config/config.yml');
+        $loader->load(__DIR__ . '/config/config.yml');
     }
 
     /**
      * {@inheritdoc}
      */
-    protected function getContainerBaseClass()
+    public function getCacheDir()
     {
-        if ('test' === $this->environment) {
-            return MockerContainer::class;
-        }
+        return sys_get_temp_dir() . '/SyliusTaxonomyBundle/cache/' . $this->getEnvironment();
```

Comment:
Won't it be better to use some common folder for that? Something like `/Sylius/TaxonomyBundle/cache/` will be easier to find and maintain. (For example, when somebody will be looking for logs)